docs/manufacturer/manufacturer-design-guidelines.mdx

```diff
@@ -265,6 +265,32 @@ Practically, this means that all pins should be on the same port, or at most two
 
   As an additional reference design, see the Fenix F405: https://oshwlab.com/jyesmith/fenix-f405
 
+### 3.2.2 Select appropriate default UARTs to avoid hijacking USB DFU 
+
+When selecting UARTs, for the default RX connection or to route to JST sockets, care should be taken to avoid those MCU pins which may interfere with USB DFU.
+
+The DFU protocol will attach to any MCU UART which is sending traffic, not only the USB DFU connection. This is a design feature of STM32 F4 MCUs and cannot be disabled.
+
+Meaning that any receiver or GPS unit attached to UART1 or UART3 (by default on F405 boards) can "hijack" DFU and make the MCU think that the serial UART should be used for DFU traffic instead of USB.
+
+Avoiding this means selecting appropriate UARTs for default connectors so the pilot will be unlikely to connect receivers or GPS units to these ports. This is mainly a problem for peripherals which are powered from the 4v5 pads and so are powered on when USB is attached.
+
+More details on STM32 Bootloaders can be found in Application Node 2606
+https://www.st.com/resource/en/application_note/an2606-stm32-microcontroller-system-memory-boot-mode-stmicroelectronics.pdf
+
+AT32 Bootloaders are described in section 2.5 at
+https://www.arterychip.com/download/DS/DS_AT32F435_437_V2.02-EN.pdf
+
+| MCU        | Pins                                        | UART                   |
+| :--------- | :----------------------------------------   | :--------------------- |
+| STM32F411  | PD5, PD6                                    | USART2                 |
+| STM32F405  | PA9, PA10, PB10, PB11                       | USART1, USART3         |
+| STM32G473  | PA9, PA10, PA2, PA3, PC10, PC11             | USART1, USART2, USART3 |
+| STM32H725  | PA9, PA10, PA2, PA3, PB10, PB11, PD8, PD9   | USART1, USART2, USART3 |
+| STM32H735  | PA9, PA10, PA2, PA3, PB10, PB11, PD8, PD9   | USART1, USART2, USART3 |
+| STM32H743  | PA9, PA10, Pb14, PB15, PA2, PA3, PB10, PB11 | USART1, USART2, USART3 |
```

Comment:
```suggestion
| STM32H743  | PA9, PA10, PB14, PB15, PA2, PA3, PB10, PB11 | USART1, USART2, USART3 |
```
Maybe use `PA2 ,` with a space for aligning pins columns

Reply:
Definitely try to align the columns. I'm not a fan of spaces before commas. If it's fixed width,the commas are probably redundant and it might look nicer with just an appropriate number of spaces.
```
| STM32G473  | PA9  PA10 PA2  PA3  PC10 PC11             | USART1 USART2 USART3 |
| STM32H725  | PA9  PA10 PA2  PA3  PB10 PB11 PD8  PD9  | USART1 USART2 USART3 |
| STM32H735  | PA9  PA10 PA2  PA3  PB10 PB11 PD8  PD9  | USART1 USART2 USART3 |
| STM32H743  | PA9  PA10 Pb14 PB15 PA2  PA3  PB10 PB11 | USART1 USART2 USART3 |
```

Reply:
Now all numbers are 2 digits so the columns align.